bokehjs/src/coffee/common/pgrid.coffee

```diff
@@ -0,0 +1,109 @@
+$ = require "jquery"
+_ = require "underscore"
+Backbone = require "backbone"
+build_views = require "./build_views"
+ContinuumView = require "./continuum_view"
+HasProperties = require "./has_properties"
+{logger} = require "./logging"
+widget = require "phosphor-widget"
+gridpanel = require "phosphor-gridpanel"
+messaging = require "phosphor-messaging"
+
+
+class PGridView extends ContinuumView
+  #className: ""
+
+  initialize: (options) ->
+    super(options)
+    window.box = this
+    @views = {}
+    
+    @panel = new gridpanel.GridPanel()
+    @panel.node.style.width = '100%'
+    @panel.node.style.height = '100%'
+    @panel.node.style.position = 'absolute'
+    @el.appendChild(@panel.node)
```

Comment:
But the panel node is attached to `PGridView`'s element, so it's only going to fill that element (assuming that element is `position: absolute | relative`).